pkg/view/employee.go

```diff
@@ -72,9 +88,14 @@ func ToEmployeeData(employee *model.Employee) *EmployeeData {
 		Gender:        employee.Gender,
 		Horoscope:     employee.Horoscope,
 		DateOfBirth:   employee.DateOfBirth,
+		GithubID:      employee.GithubID,
+		DiscordID:     employee.DiscordID,
+		NotionID:      employee.NotionID,
 		WorkingStatus: employee.WorkingStatus,
 		JoinedDate:    employee.JoinedDate,
 		LeftDate:      employee.LeftDate,
+		Projects:      projects,
```

Comment:
We don't need full project data here. Only need projectID and its name

Reply:
updated